README.md

```diff
@@ -164,14 +164,18 @@ no value for the key.
 
 ```ruby
 class Book
-  STATES = NxtSupport::EnumHash['draft', 'revised', 'Published', 'in weird State']
+  STATES = NxtSupport::EnumCollection['draft', 'revised', 'Published', 'in weird State']
```

Comment:
Why `Collection`? Isn't this implicitly expressed by `Enum`?

Reply:
Well I've already used `Enum` as a class. Could be `Enums` I guess.

Reply:
@nsommer Can we proceed with this?